Packs/TwitterIOCHunter-FullDailyFeed/Integrations/TwitterIOCHunterFullDailyFeed/TwitterIOCHunterFullDailyFeed.py

```diff
@@ -29,42 +33,61 @@ def find_type_and_value(indicatordata):
         return 'Error', ''
 
 
-def get_indicators_command(client):
+def get_indicators_command(client, url, feed_tags=None, tlp_color=None):
     listofindicators = []
-    result = client._http_request('GET', '/')
+    result = client._http_request('GET', full_url=url)
     for item in result:
         typeofindicator, valueofindicator = find_type_and_value(item)
         for newitem in valueofindicator:
             data = {'type': typeofindicator,
                     'value': newitem,
                     'service': 'Twitter IOC Hunter',
-                    'rawJSON': {'reference': item.get('reference'), 'tweet': item.get('tweet')},
+                    'fields': {
+                        'firstseenbysource': item.get('tweet').get('timestamp'),
+                        'tags': feed_tags,
+                        'tweet': item.get('tweet').get('tweet'),
```

Comment:
This (tweet) Indicator field does not currently exist, so there is no meaning to this mapping.
You can create one like this and add it to your Pack.

Reply:
Makes sense. I will remove it.